### PR TITLE
docs: Top Games on Dusk

### PR DESCRIPTION
**Pull request by mieszko4**
* Add new page to show top 10 games on Dusk for last completed week based on playtime
* Use TypeScript for implementation

Tested on Chrome, Chrome Android, Safari, Safari iOS

### CODE REVIEW

File: docs/src/components/TopGamesOnDusk/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+    line-height: 36px;
+    letter-spacing: -0.02em;
+
+    color: #eac6ff;
+  }
+
+  .previewImg {
+    order: 2;
+
+    // Devide original dimensions (104x104) by 4 and then scale it up by 4
```
Comment (Gongreg) [inline docs/src/components/TopGamesOnDusk/styles.module.scss]:
```suggestion
    // Divide original dimensions (104x104) by 4 and then scale it up by 4
```

File: docs/src/components/TopGamesOnDusk/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+    gap: 190px;
+
+    margin-top: 190px;
+  }
+
+  .topGame {
+    max-width: 300px;
+    padding-bottom: 20px;
+
+    &:global(.place-1) {
```
Comment (Gongreg) [inline docs/src/components/TopGamesOnDusk/styles.module.scss]: Not sure why we need a global class for it.

**Review by Gongreg — APPROVED** [review]

File: docs/src/components/TopGamesOnDusk/styles.module.scss (hunk 2 above)
Comment (mieszko4) [reply docs/src/components/TopGamesOnDusk/styles.module.scss]: I dynamically generate the class name based on `place` prop.
It is to simplify code in react component.

Comment (mieszko4) [reply docs/src/components/TopGamesOnDusk/styles.module.scss]: Just realized I can do it without global. Done in https://github.com/dusk-gg/dusk/pull/527/commits/3647058a4218bba982860aa969c50400a54a0cb4

**Pull request merged**